app/assets/stylesheets/templates/_homepage.scss

```diff
@@ -1,4 +1,25 @@
 .template--homepage {
+  .trust-badge {
```

Comment:
NOTE: all CSS in this PR was taken directly from GCF (or adapted slightly as needed)